addons/html_builder/static/src/plugins/background_option/background_position_overlay.xml

```diff
@@ -4,13 +4,11 @@
 <t t-name="html_builder.BackgroundPositionOverlay">
     <div class="o_we_background_position_overlay" t-ref="backgroundOverlay">
         <div class="o_we_overlay_content position-absolute" t-ref="overlayContent">
-            <div class="o_overlay_background" t-ref="overlayBackground">
-                <div t-on-mousedown.prevent="onDragBackgroundStart"
-                    t-ref="parentBgDragger"
-                    title="Click and drag the background to adjust its position!"
-                    >
-                    <t t-out="props.outerHtmlEditingElement"/>
-                </div>
+            <div class="o_overlay_background" t-on-mousedown.prevent="onDragBackgroundStart"
```

Comment:
One layer of div was too much and was an issue when trying to change the background position of an image with a parallax